### PR TITLE
fix(chat): mobile adaptation of chat toolbar

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
fix mobile adaptation of chat toolbar

**Which issue(s) this PR fixes** 🔨
[AP-840](https://satellite-im.atlassian.net/browse/AP-840)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by phillsatellite** [comment]
Just want to confirm, I saw ticket AP-840 was for the Search Bar getting cut off, but when I tested Search bar is gone and when user clicks the Satellite icon at the top right nothing happens, is this intended?
IMG_4662

**Comment by AlexZakablukov** [comment]
I made changes depand on this figma file, is it correct ? https://www.figma.com/file/AwWgkAtjRDfVJlxpMbBbgP/Messenger?node-id=88%3A2789

**Comment by phillsatellite** [comment]
> I made changes depend on this figma file, is it correct ? https://www.figma.com/file/AwWgkAtjRDfVJlxpMbBbgP/Messenger?node-id=88%3A2789

Reached out to Liz and it is correct! She suggested one thing we could add is having the users profile modal pop up similar to web version 👍

**Comment by AlexZakablukov** [comment]
> > I made changes depend on this figma file, is it correct ? https://www.figma.com/file/AwWgkAtjRDfVJlxpMbBbgP/Messenger?node-id=88%3A2789
> 
> Reached out to Liz and it is correct! She suggested one thing we could add is having the users profile modal pop up similar to web version 👍

**Comment by AlexZakablukov** [comment]
In web version you also can`t to click on users you messaging with

**Comment by AlexZakablukov** [comment]
I mean this one 
image

File: components/views/chat/group/Group.vue [hunk 1]
```diff
@@ -80,27 +78,11 @@ export default Vue.extend({
...
-      if (!this.ui.quickProfile) {
-        openQuickProfile()
-        return
-      }
-
-      setTimeout(() => {
-        if (!this.ui.quickProfile) {
-          openQuickProfile()
-        }
-      }, 0)
```
Comment (molimauro) [inline components/views/chat/group/Group.vue:103]: This logic is missing in the refactor. Without this you have this bug:

https://user-images.githubusercontent.com/50131258/155996301-5fb44c3b-88e6-4209-abfd-3100cb39bd68.mov

Comment (AlexZakablukov) [reply components/views/chat/group/Group.vue]: changed

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**